ObservatoryCore/UI/Views/BasicUIView.axaml.cs

```diff
@@ -164,10 +193,164 @@ private Grid GenerateCoreUI()
                 Properties.Core.Default.NativeNotify = false;
                 Properties.Core.Default.Save();
             };
+            
+            Button notifyTestButton = new()
+            { 
+                Content = "Test",
+                HorizontalAlignment = Avalonia.Layout.HorizontalAlignment.Right
+            };
 
-            corePanel.AddControl(nativeNotifyCheckbox, rowTracker.NextIndex(), 0, 2);
+            notifyTestButton.Click += (object sender, RoutedEventArgs e) =>
+            {
+                Avalonia.Threading.Dispatcher.UIThread.InvokeAsync(() =>
+                {
+                    var notifyWindow = new UI.Views.NotificationView() { DataContext = new UI.ViewModels.NotificationViewModel("Test Notification", "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras suscipit hendrerit libero ac scelerisque.") };
+                    notifyWindow.Show();
+                });
+            };
 
-            #endregion
+            TextBlock notifyFontLabel = new() 
+            { 
+                Text = "Font: ",
+                HorizontalAlignment = Avalonia.Layout.HorizontalAlignment.Right,
+                VerticalAlignment = Avalonia.Layout.VerticalAlignment.Center
+            };
+            ComboBox notifyFontDropDown = new()
+            {
+                MinWidth = 200
+            };
+
+            notifyFontDropDown.Items = new System.Drawing.Text.InstalledFontCollection().Families.Select(font => font.Name);
+
+            if (Properties.Core.Default.NativeNotifyFont.Length > 0)
+            {
+                notifyFontDropDown.SelectedItem = Properties.Core.Default.NativeNotifyFont;
+            }
+
+            notifyFontDropDown.SelectionChanged += (object sender, SelectionChangedEventArgs e) =>
+            {
+                var comboBox = (ComboBox)sender;
+                Properties.Core.Default.NativeNotifyFont = comboBox.SelectedItem.ToString();
+                Properties.Core.Default.Save();
+            };
+            
+            TextBlock monitorLabel = new()
+            {
+                Text = "Display: ",
+                HorizontalAlignment = Avalonia.Layout.HorizontalAlignment.Right,
+                VerticalAlignment = Avalonia.Layout.VerticalAlignment.Center
+            };
+            ComboBox monitorDropDown = new()
+            { 
+                MinWidth = 200
+            };
+
+            List<string> displays = new();
+            displays.Add("Primary");
+            var screens = ((Window)Parent.Parent.Parent.Parent).Screens.All;
```

Comment:
No action required: Is there a safer way of fetching Window? If something gets inserted into the tree, I assume this could break the displays list.

Reply:
Any changes to the tree would be under the control of the project so that's not something that would come out of the blue. That said, could iterate up the tree until the type is window to be extra safe. Might do that when I get home this evening